tests/ci/create-osbuild.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+set -euo pipefail
+
+# following steps as noted on https://hackmd.io/ZbUOkeIXTjaP7XVpTSsrNw?view
+
+# Get OS data.
+source /etc/os-release
+
+ID=${ID:-}
+ARCH=$(arch)
+OS_VARIANT=${OS_VARIANT:-}
+IMAGE_TYPE=${IMAGE_TYPE:-}
+UUID=${UUID:-local}
+BOOT_LOCATION="http://mirror.centos.org/centos/8-stream/BaseOS/${ARCH}/os/"
+
+
+# install osbuild and osbuild-tools, which contains osbuild-mpp utility
+dnf -y copr enable @osbuild/osbuild
+SEARCH_PATTERN='baseurl=https://download.copr.fedorainfracloud.org/results/@osbuild/osbuild/epel-8-\$basearch/'
+REPLACE_PATTERN='baseurl=https://download.copr.fedorainfracloud.org/results/@osbuild/osbuild/centos-stream-8-$basearch/'
+sed -i -e "s|$SEARCH_PATTERN|$REPLACE_PATTERN|" \
+	/etc/yum.repos.d/_copr\:copr.fedorainfracloud.org\:group_osbuild\:osbuild.repo
+dnf -y install osbuild osbuild-tools
```

Comment:
I wonder if we're missing dependencies. We're running the tests in a Beaker machine that has already run the other build (`osbuild-composer`) and that has installed its dependencies.
We should try in a brand new machine, so we're sure that we aren't missing dependencies.